test/alternator/test_scan.py

```diff
@@ -414,29 +414,38 @@ def test_scan_long_partition_tombstone_string(dynamodb, query_tombstone_page_lim
 # single returned page is still limited. DynamoDB specifies it should be
 # limited to 1 MB. In Alternator the limit is close to 1 MB, but it turns
 # out (see issue #10327) that for small tables the page size can grow up
-# to 3 MB. The following test accepts this as ok. Note that for larger tables,
-# the page size goes back to being closer to 1 MB.
+# signifficantly. The following test accepts this as ok. Note that for larger
+# tables, the page size goes back to being closer to 1 MB.
 #
 # This test is for Scan paging on a table with many small partitions. We have
 # a separate test for a Query over a single long partition with many rows -
 # test_query.py::test_query_reverse_longish (the test's name suggests it
 # checks reverse queries, but it also checks the unreversed unlimited query).
 # For single-partition scans, the page size is more exactly 1 MB.
-def test_scan_paging_missing_limit(dynamodb):
+#
+# The test is run twice, with two different values of N (the number of items in a batch write).
```

Comment:
The new comment and the commit message mix up "page size", "batch size" and even uses the phrase "batch write" for the table size (note that the AWS SDK does **not** write the entire thing in a single batch, it's just a loop which collects several smaller batches). I'll fix it up myself.